ccmlib/common.py

```diff
@@ -76,6 +77,36 @@ class UnavailableSocketError(CCMError):
     pass
 
 
+class LogPatternToVersion(object):
+
+    def __init__(self, versions_to_patterns, default_pattern):
+        self.versions_to_patterns = versions_to_patterns
+        if "0" not in self.versions_to_patterns:
```

Comment:
Here I'm working on the assumption someone might add a version "0" pattern, though it seems... unlikely.